templates/quickstart-hashicorp-consul-master.template

```diff
@@ -286,7 +293,7 @@ Resources:
       TemplateURL: !Sub >-
         https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-linux-bastion/templates/linux-bastion.template
       Parameters:
-        BastionAMIOS: Ubuntu-Server-20.04-LTS-HVM
+        BastionAMIOS: Amazon-Linux2-HVM
```

Comment:
Since this isn't strictly required, and I believe HashiCorp's OS preference is ubuntu, can we revert these?